promoter/skyd.go

```diff
@@ -54,3 +58,43 @@ func (p *Promoter) staticWatchedSkydAddresses() ([]types.UnlockHash, error) {
 	}
 	return wag.Addresses, nil
 }
+
+// staticTxnsByAddress fetches all confirmed transactions for a given address
+// from skyd and returns them as an interface slice ready to be inserted into
+// the database.
+func (p *Promoter) staticTxnsByAddress(addr types.UnlockHash) ([]interface{}, error) {
```

Comment:
Why do we return `[]interface{}` and not `[]Transaction`?

Reply:
Because we insert them into the database later. For which we need a `[]interface{}` because Go can't cast between a slice of a type and a slice of an interface.

Reply:
*facepalm*
I understand...